core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/GraphicElementFactory.java

```diff
@@ -207,12 +246,12 @@ public static String getCSSOfLook(String lookName) {
   }
 
   public static SettingBundle getLookSettings(String lookName) {
-    String selectedLook = lookSettings.getString(lookName, null);
-    if (selectedLook == null) {
-      // ce look n'existe plus, look par defaut
-      selectedLook = defaultLook;
+    String look = lookSettings.getString(lookName, null);
+    if (!isLookExist(lookName)) {
+      lookName = getExistingLook();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "lookName". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1226)